notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings.ipynb

```diff
@@ -105,305 +108,252 @@
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "hhq5zEbGg0XX"
-      },
-      "source": [
-        "### Colab only: Uncomment the following cell to restart the kernel."
+        "### Initialize Vertex AI SDK for Python\n",
+        "\n",
+        "To get started using Vertex AI, you must have an existing Google Cloud project and [enable the Vertex AI API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com). "
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "EzrelQZ22IZj"
+        "id": "b519c49740cb"
       },
       "outputs": [],
       "source": [
-        "# Automatically restart kernel after installs so that your environment can access the new packages\n",
-        "# import IPython\n",
+        "from google.cloud import aiplatform\n",
         "\n",
-        "# app = IPython.Application.instance()\n",
-        "# app.kernel.do_shutdown(True)"
+        "aiplatform.init(project=PROJECT_ID, location=LOCATION, staging_bucket=BUCKET_URI)"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "zgPO1eR3CYjk"
+        "id": "44b8dbd3f108"
       },
       "source": [
-        "### Create a Cloud Storage bucket\n",
-        "\n",
-        "Create a storage bucket to store intermediate artifacts such as datasets."
+        "### Import the required libraries"
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "MzGDU7TWdts_"
+        "id": "bf6f53cd87ee"
       },
       "outputs": [],
       "source": [
-        "BUCKET_URI = f\"gs://your-bucket-name-{PROJECT_ID}-unique\"  # @param {type:\"string\"}"
+        "import json\n",
+        "import tempfile\n",
+        "from typing import List\n",
+        "\n",
+        "import numpy as np\n",
+        "import redis\n",
+        "import tensorflow as tf\n",
+        "import tensorflow_hub as hub\n",
+        "# Registers the ops.\n",
+        "import tensorflow_text as text  # noqa: F401\n",
+        "from google.cloud import bigquery\n",
+        "from tqdm.auto import tqdm"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "-EcIXiGsCePi"
-      },
-      "source": [
-        "**If your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "NIq7R4HZCfIc"
+        "id": "lR6Wwv-hCCN-"
       },
-      "outputs": [],
       "source": [
-        "! gsutil mb -l {REGION} -p {PROJECT_ID} {BUCKET_URI}"
+        "## Prepare the data\n",
+        "\n",
+        "For this tutorial, you use the [Stack Overflow dataset](https://console.cloud.google.com/marketplace/product/stack-exchange/stack-overflow) of question and answers hosted on BigQuery.\n",
```

Comment:
Don't need "you". Note that "use" is a directive and implies "you," the reader of the document.
- "For this tutorial, use the..."